src/module/actor/character/sheet.ts

```diff
@@ -746,37 +748,37 @@ class CharacterSheetPF2e extends CreatureSheetPF2e<CharacterPF2e> {
 
         $formulas.find(".daily-crafting").on("click", async () => await this.actor.performDailyCrafting());
 
-        PCSheetTabManager.initialize(this.actor, $html.find<HTMLAnchorElement>('a[data-action="manage-tabs"]')[0]);
+        PCSheetTabManager.initialize(this.actor, $html.find<HTMLAnchorElement>("a[data-action=manage-tabs]")[0]);
 
         // Feat Browser shortcut links
-        $html.find(".feat-browse").on("click", (event) => this.onClickBrowseFeatCompendia(event));
+        for (const link of Array.from(html.querySelectorAll<HTMLAnchorElement>(".feat-browse"))) {
+            link.addEventListener("click", this.onClickBrowseFeatCompendia);
+        }
```

Comment:
I'm starting to see why jquery still exists. DOM api is really tedious, and node list in general are kinda painful. Is there any lightweight lib you're aware of to not deal with all this garbage?

Reply:
we could acquiesce to the `NodeList#forEach`